Readability.js

```diff
@@ -680,6 +688,7 @@ Readability.prototype = {
         if (stripUnlikelyCandidates) {
           if (this.REGEXPS.unlikelyCandidates.test(matchString) &&
               !this.REGEXPS.okMaybeItsACandidate.test(matchString) &&
+              !this._isMaybeAncestorACandidate(node, 3) &&
```

Comment:
I think this is just set to 3 because it always was and attempts to score all the ancestors with incremental backoff produced regressions. Magical constants are sad, but I don't know that we can easily remove them here.